<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-09-12T13:12:39Z" name="Default suite" finished-at="2020-09-12T13:12:52Z" duration-ms="12968">
    <groups>
    </groups>
    <test started-at="2020-09-12T13:12:39Z" name="Default test" finished-at="2020-09-12T13:12:52Z" duration-ms="12968">
      <class name="WebDrivCheck.webCheck">
        <test-method is-config="true" signature="setUp()[pri:0, instance:WebDrivCheck.webCheck@762ef0ea]" started-at="2020-09-12T13:12:39Z" name="setUp" finished-at="2020-09-12T13:12:47Z" duration-ms="8771" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="logoPresence()[pri:1, instance:WebDrivCheck.webCheck@762ef0ea]" started-at="2020-09-12T13:12:47Z" name="logoPresence" description="Verify Logo presence on Home Page TestNG annotation" finished-at="2020-09-12T13:12:48Z" duration-ms="158" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoPresence -->
        <test-method signature="loginTest()[pri:2, instance:WebDrivCheck.webCheck@762ef0ea]" started-at="2020-09-12T13:12:48Z" name="loginTest" finished-at="2020-09-12T13:12:51Z" duration-ms="3139" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [noCommerce demo store] but found [nopCommerce demo store. Login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [noCommerce demo store] but found [nopCommerce demo store. Login]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at WebDrivCheck.webCheck.loginTest(webCheck.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method signature="registrationTest()[pri:3, instance:WebDrivCheck.webCheck@762ef0ea]" started-at="2020-09-12T13:12:51Z" name="registrationTest" description="Verify Registration form TestNG Test" finished-at="2020-09-12T13:12:51Z" duration-ms="6" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping this test]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping this test
at WebDrivCheck.webCheck.registrationTest(webCheck.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrationTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:WebDrivCheck.webCheck@762ef0ea]" started-at="2020-09-12T13:12:51Z" name="tearDown" finished-at="2020-09-12T13:12:52Z" duration-ms="844" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- WebDrivCheck.webCheck -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
